styles/application.css

```diff
@@ -1,57 +1,2 @@
 @import 'prism.css';
 
-/* custom properties */
```

Comment:
These CSS rules have never been preprocessed into the target CSS directory, or included in a gem. A good example of confusion in the project.

They seem to be a work in progress as the break stuff in the style-guide interface. Let's remove them for now so we have a working build pipeline.